app/build.gradle

```diff
@@ -179,6 +179,7 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
     kotlinOptions.freeCompilerArgs += [
             '-Xuse-experimental=kotlinx.coroutines.ExperimentalCoroutinesApi',
             '-Xuse-experimental=kotlinx.coroutines.FlowPreview',
-            '-Xuse-experimental=kotlin.time.ExperimentalTime'
+            '-Xuse-experimental=kotlin.time.ExperimentalTime',
+            '-Xuse-experimental=io.ktor.util.KtorExperimentalAPI'
```

Comment:
Is there anything we can do in the PoWeb library to prevent devs from having to do this? I couldn't find the answer online

Reply:
I'm not sure, because part of Ktor is exposed to us. But I can look into it.

Reply:
Thanks. I'd imagine this is the culprit: https://github.com/relaycorp/relaynet-poweb-jvm/blob/0fcb768994cc6b9f287e371bdfd9c617041b8330/src/main/kotlin/tech/relaycorp/poweb/PoWebClient.kt#L53

But I don't know how to prevent it from propagating to clients of the PoWeb lib.

Reply:
@gnarea wouldn't the same compilerArgs approach I did work for the library? But I will look this up.

Reply:
Thanks for looking into this! Can you please use the new version of PoWeb so we can use the fix in https://github.com/relaycorp/relaynet-poweb-jvm/pull/62 and undo this workaround?